sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Customized/Models/UpdateModelSettingsOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Iot.TimeSeriesInsights.Customized
+{
+    /// <summary>
+    /// Request to update model settings. One of &quot;name&quot; or &quot;defaultTypeId&quot; must be set.
+    /// </summary>
+    [CodeGenModel("UpdateModelSettingsRequest")]
```

Comment:
changing visibility of this class to public 

If was internal initially it shows CS0051: inconsistent accessibility where the method UpdateModelSettings is public